### ISSUE TITLE
gradient_accumulation_batch_size missing in trainer

### ISSUE THREAD

**Issue by knighter** [allenai/scibert#65]
I found 
`gradient_accumulation_batch_size` exists in several scibert conf, such as 
https://github.com/allenai/scibert/blob/8562a120e6788dcbadbe05ef7fd4463dee17ee59/allennlp_config/ner.json
but allennlp trainer doesn't have this param,
https://github.com/allenai/allennlp/blob/master/allennlp/training/trainer.py

**Comment by ibeltagy** [comment]
Yes, AllenNLP doesn't support gradient accumulation. We have it implemented in our fork of allennlp (check requirements: https://github.com/allenai/scibert/blob/master/requirements.txt)

**Comment by knighter** [comment]
got it, thank you!

**Comment by xiaoruijiang** [comment]
Hi, The installation command in the readme returned the following "allennlp version not found error". Can I ask what is the right way to install the tool and reproduce the results? 

Thank you very much!

Obtaining allennlp from git+git://github.com/ibeltagy/allennlp@fp16_and_others#egg=allennlp (from -r requirements.txt (line 1))
WARNING: Discarding git+git://github.com/ibeltagy/allennlp@fp16_and_others#egg=allennlp. Command errored out with exit status 128: git rev-parse HEAD Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement allennlp (unavailable)
ERROR: No matching distribution found for allennlp (unavailable)
